model/schema/highThroughputExpression.yaml

```diff
@@ -0,0 +1,578 @@
+          When using the "summary" note type, this is the free text summary or description of the dataset
+      references:
+        description: >-
+          The list of publications associated with this dataset
+      title:
+        description: >-
+          The free text title of the dataset
+
+  HighThroughputExpressionDatasetAnnotationDTO:
+    is_a: SubmittedObjectDTO
+    description: >-
+      A DTO entry for a published high-throughput expression dataset annotation.
+    slots:
+      - name # title in agr_schemas json
+      - reference_curies # publications in agr_schemas json
+      - note_dto # use 'summary' note type; summary in agr_schemas json
+      - number_of_channels # numChannels in agr_schemas json
+      - sub_series_dto # subSeries in agr_schemas json
+      - category_tag_names # categoryTags in agr_schemas json
+    slot_usage:
+      related_note:
+        description: >-
+          When using the "summary" note type, this is the free text summary or description of the dataset
+      references:
+        description: >-
+          The list of publications associated with this dataset
+      title:
+        description: >-
+          The free text title of the dataset
+
+  HighThroughputExpressionDatasetSampleAnnotation:
+    is_a: SubmittedObject
+    description: >-
+     An entry with dataset sample object associations.
+    slots:
+      - sample_id # sampleId in agr_schemas json
+      - sample_title # sampleTitle in agr_schemas json
+      - sample_type # sampleType in agr_schemas json
+      - sample_age # sampleAge in agr_schemas json
+      - sample_locations # sampleLocations in agr_schemas json
+      - where_expressed_statement # whereExpressedStatement in whereExpressed.json
+      - abundance # abundance in agr_schemas json
+      - genomic_information # genomicInformation in agr_schemas json
+      - taxon # taxonId in agr_schemas json
+      - genetic_sex # sex in agr_schemas json
+      - expression_assay_used # assayType in agr_schemas json
+      - sequencing_format # sequencingFormat in agr_schemas json
+      - assembly_versions # assemblyVersions in agr_schemas json
+      - related_notes # notes in agr_schemas json
+      - dataset_ids # datasetIds in agr_schemas json
+      - microarray_sample_details # microarraySampleDetails in agr_schemas json
+      - when_expressed_stage_name # biosampleAge.json > stage > when_expressed.json > stageName in agr_schemas json
+    slot_usage:
+      sample_id:
+        description: >-
+          The ID for this sample object, MOD or external database ID (ex: GEO);
+          primaryId and secondaryId
+      genetic_sex:
+        description: >-
+          The sex of the organism the sample came from
+      expression_assay_used:
+        description: >-
+          ID of the type of assay used for this sample; sample or experiment property? See enumeration below.
+        range: high_throughput_expression_assay_values
+    rules:
+      - postconditions:
```

Comment:
I think it would be clearer to just make `datasets_ids`, `expression_assay_used`, and `sample_type` all required fields and then just have this post-condition to indicate that either `mod_entity_id` or `sample_title` is required

Reply:
Would `mod_internal_id` also be valid in the absence of `mod_entity_id` or `sample_title`?

Reply:
Maybe? I would have to ask curators if this would ever apply